scripts/go_test.sh

```diff
@@ -3,4 +3,4 @@
 set -o pipefail
 
 # Run go test and colorize output (PASS - green, FAIL - red).
-go test -v -coverprofile=coverage.txt -covermode=atomic ./... | sed ''/PASS/s//$(printf "\033[32mPASS\033[0m")/'' | sed ''/FAIL/s//$(printf "\033[31mFAIL\033[0m")/''
+go test -v -p 1 -coverprofile=coverage.txt -covermode=atomic ./... | sed ''/PASS/s//$(printf "\033[32mPASS\033[0m")/'' | sed ''/FAIL/s//$(printf "\033[31mFAIL\033[0m")/''
```

Comment:
can you add a comment explaining why the `-p 1` is needed? otherwise someone may remove it and re-introduce the intermittent errors

Reply:
yep, done